src/systemctl/systemctl.c

```diff
@@ -116,7 +116,7 @@ static bool arg_no_wall = false;
 static bool arg_no_reload = false;
 static bool arg_value = false;
 static bool arg_show_types = false;
-static bool arg_ignore_inhibitors = false;
+static const char *arg_respect_inhibitors = "auto";
```

Comment:
nah, please parse the data once, and store it parsed. Do not store the original strings. for this value consider using a tristate, i.e. a simple int, where < 0 means "auto", > 0 means "yes" and == 0 means "no". We use such tristates (i.e. booleans with an extra value) quite a lot.

Reply:
I choose to set use "no" < 0, "auto" = 0 and "yes" > 0. I hope this is okay but it seemed way better because "auto" is kind of something between "no" and "yes" because then it depends on if the invocation was interactive or not.